test/data/manifests/rhel_88-x86_64-ec2_sap-boot.json

```diff
@@ -3831,9 +3829,6 @@
           "sha256:09149617095dc52e19cdce1e45c8245e1e92d371bd4d107320ff56788b9977f1": {
             "url": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-baseos-n8.8-20221025/Packages/libusbx-1.0.23-4.el8.x86_64.rpm"
           },
-          "sha256:0942e19ecae75b9b0f30f10c1469b783e0887728bdbd946b6e49d7c3fabc4593": {
-            "url": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-appstream-n8.8-20221025/Packages/redhat-cloud-client-configuration-1-10.el8.noarch.rpm"
-          },
```

Comment:
Same as 8.7, the package must be installed. See https://github.com/osbuild/osbuild-composer/pull/3081

Reply:
Ok, figured these out. One of the times this was rebase back onto main didn't notice that `rhelEc2CommonPackageSet` was being used. Not sure if this was my fault or git's (I don't remember any merge conflicts). Should be all fixed now.